pkg/support/dump_windows_test.go

```diff
@@ -0,0 +1,102 @@
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package support
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	exectesting "k8s.io/utils/exec/testing"
+
+	"github.com/spf13/afero"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"k8s.io/utils/exec"
+)
+
+type testExec struct {
+	exectesting.FakeExec
+}
+
+func (te *testExec) Command(cmd string, args ...string) exec.Cmd {
+	fakeCmd := new(exectesting.FakeCmd)
+	fakeCmd.CombinedOutputScript = append(fakeCmd.CombinedOutputScript, func() ([]byte, []byte, error) {
+		return []byte(fmt.Sprintf("%s %s", cmd, strings.Join(args, " "))), nil, nil
+	})
+	return fakeCmd
+}
```

Comment:
are we defining this twice?

that could explain why your tests are failing on Windows